scripts/fuel-tx.ts

```diff
@@ -28,11 +35,11 @@ async function run() {
   const provider = new Provider("http://127.0.0.1:4000/graphql");
   const from = "0x0101010101010101010101010101010101010101010101010101010101010101";
   const to = genBytes32();
-  const amount = BigNumber.from(1);
+  const amount = thousandSeparator(BigNumber.from(1));
   await provider.sendTransaction({
     type: TransactionType.Script,
-    gasPrice: BigNumber.from(0),
-    gasLimit: BigNumber.from(1000000),
+    gasPrice: thousandSeparator(BigNumber.from(0)),
+    gasLimit: thousandSeparator(BigNumber.from(1000000)),
```

Comment:
This doesn't quite seem right? You shouldn't be adding separators when _sending_ a tx, that should just be the number. It's only _displaying_ the numbers that should involve separators.

Reply:
oh i see, which page displays the numbers.

Reply:
You can use something like this: https://github.com/FuelLabs/swayswap-demo/blob/415af2b3ce6bc49242741ac0d7e66efe59bdecc6/client/src/components/Assets.tsx#L80

Reply:
Cool, I Think I'll use this, Thank you.